tests/validator_test.go

```diff
@@ -271,6 +326,34 @@ var _ = Describe("Template validator", func() {
 					podSpec.Tolerations == nil
 			}, timeout, 1*time.Second).Should(BeTrue())
 		})
+
+		// TODO - this test may fail if the operator is too slow and does not notice Deployment state when
+		//        where not all pods are running
+		It("[test_id: TODO]should set available condition when at least one validator pod is running", func() {
+			watch, err := StartWatch(sspListerWatcher)
```

Comment:
I would prefer if this kind of test was `Pending()` instead of `It()` until we figure out how to properly test it or remove it

Reply:
Done